services/MerchStoreService.ts

```diff
@@ -185,7 +209,7 @@ export default class MerchStoreService {
     const order = await this.transactions.readOnly(async (txn) => Repositories
       .merchOrder(txn)
       .findByUuid(uuid));
-    if (!order) throw new NotFoundError();
+    if (!order) throw new NotFoundError(NotFoundErrors.MERCH_ORDER);
```

Comment:
Can we make these changes in another PR?

Reply:
Sure we could - I initially refactored them because a lot of the error messages were duplicated, but it just seems weird having one service with canned error messages and others without. Plus this PR is pretty sizeable on its own.